### ISSUE TITLE
User reflections cannot be edited

### ISSUE THREAD

**Issue by kytrinyx** [exercism/exercism#3894]
_Imported from https://github.com/exercism/-DEPRECATED-PLEASE-USE-exercism.io-REPO-v2-feedback/issues/213._
_Originally posted by @carolineschnapp._

#

User reflections cannot be edited. I found a way which is hackish:

1. Copy your old reflection to the clipboard
2. Go to the track, locate your exercise
3. Click the "Complete Exercise (Unapproved)" button, always clickable even when your solution was already completed. 
4. Paste your reflection, and edit it.

In other words, it's possible to complete an already completed exercise, then you get another chance at submitting a User reflection.

**Comment by guillearch** [comment]
I tried this to fix a typo in a reflection, but I couldn't find the "Complete Exercise". Is there another way to edit the reflection of a submitted solution?

**Comment by F3PiX** [comment]
Currently, no, sorry. The 'complete exercise' button described above is no longer available AFAIK.

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂
